x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/error_callout.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiCallOut } from '@elastic/eui';
+
+interface Props {
+  error: string;
+}
+
+export const ErrorCallout: FC<Props> = ({ error }) => {
+  let errorCallout = (
+    <EuiCallOut
+      title={i18n.translate('xpack.ml.dataframe.analytics.regressionExploration.generalError', {
+        defaultMessage: 'An error occurred loading the data.',
+      })}
+      color="danger"
+      iconType="cross"
+    >
+      <p>{error}</p>
+    </EuiCallOut>
+  );
+  // Job was created but not started so the destination index has not been created
+  if (error.includes('index_not_found')) {
+    errorCallout = (
+      <EuiCallOut
+        title={i18n.translate('xpack.ml.dataframe.analytics.regressionExploration.evaluateError', {
+          defaultMessage: 'An error occurred loading the data.',
+        })}
+        color="danger"
+        iconType="cross"
+      >
+        <p>
+          {i18n.translate('xpack.ml.dataframe.analytics.regressionExploration.noIndexCalloutBody', {
+            defaultMessage:
+              'The query for the index returned no results. Please make sure the destination index exists and contains documents.',
+          })}
+        </p>
+      </EuiCallOut>
+    );
+  } else if (error.includes('No documents found')) {
+    // Job was started but no results have been written yet
+    errorCallout = (
+      <EuiCallOut
+        title={i18n.translate(
+          'xpack.ml.dataframe.analytics.regressionExploration.noDataCalloutTitle',
+          {
+            defaultMessage: 'Empty index query result.',
+          }
+        )}
+        color="primary"
+      >
+        <p>
+          {i18n.translate('xpack.ml.dataframe.analytics.regressionExploration.noDataCalloutBody', {
+            defaultMessage:
+              'The query for the index returned no results. Please make sure the job has been started and the index contains documents.',
```

Comment:
Can the evaluate API only be run against jobs that are complete? If so, maybe reword the text to say `Please make sure the job has been completed and ...`

Reply:
Good idea - reworded in 6de9d0d162db3f4225bd32b347bdd502f4d51719